Dockerfile

```diff
@@ -19,6 +19,7 @@ COPY controller/ controller/
 COPY cloud/ cloud/
 COPY util/ util/
 COPY version/ version/
+COPY mock/ mock/
```

Comment:
do we need to add this just for testing or is it also needed while the app is running?

Reply:
I was getting errors with the build since I'm importing `mock` in `cloud/scope` to avoid a circular dependency. But I think I can get around that so we don't need to include it in the build.